docs/community/working-group-definitions.md

```diff
@@ -0,0 +1,195 @@
+# Working Group Defintions
```

Comment:
The only doubt about group members is that we see the activity of some members never or very rarely.

Reply:
Are there already community candidates for these groups? If not, why start all this?

I'm amazed  surprised that MSFT won't bring its partners here in any way.

Reply:
@mklement0: you raise a lot of really good points, did my best to address them all below, but let me know if there are still some lingering.

First, regarding whether PS Team members would ever be *removed* from WGs: obviously, if someone leaves the company or the PowerShell Team, they would probably not want to spend time as a WG member anymore. But even in other cases, I don't think it should be entirely ruled out: folks shuffle around their interests and roles within the team somewhat fluidly, and based on other time commitments, folks may need to leave WGs. 

(However, given the nature of certain internal requirements, I do think we'll need all WGs to have at least some Microsoft representation.)

But a lot of the language here is intended to convey that being a WG member is a *time commitment*. WG members are expected to be actively contributing on a regular basis, and should not be occasional passive advisors. In that sense, and in the sense that WGs have formal authority, everyone on the WG should be more than a traditional "fellow community member", and may constitute the equivalent of "job responsibilities" for folks even outside of Microsoft. 

Part of the motivation behind not adding community members on day one is that we would like to experiment with different engagement models. We don't yet know how engagement scales, how WGs want to reach consensus, etc., and we'd rather not add a bunch of folks to WGs only to realize that they're making ultimately undesirable decisions that need to be rolled back. Similarly, we want WGs to use this experimental timeframe to establish principles and criteria by which future decisions get made. 

As for driving towards consensus: the language there is intended to specifically discourage blanket "vetos" or voting that traipses over the concerns of a minority of members. In other words, folks shouldn't just be saying "the 4 of us think you're wrong, so no", but rather convincing other members that their position is the correct one.

From those perspectives, WGs are not really supposed to be a sum of individuals with completely random interests and opinions, but rather a body that generally represents the overall project direction and philosophy. For example, we don't expect WGs to suddenly want to make breaking changes that wouldn't have historically been accepted so far. 

I hope that clarifies some of your concerns. I'm not sure that all of this necessarily belongs in this set of docs, but I'm perfectly happy to make targeted clarifications or flesh out more of the specifics where you think it makes sense.

Reply:
@joeyaiello, I think you meant to direct your comment to @vexx32.

Reply:
Thanks, @mklement0 😁 

Thanks @joeyaiello! I think that all makes sense. I'm not sure whether some of that should be in this doc either, to be honest, but the additional clarity is nonetheless appreciated. 💖 😊